completions/ant

```diff
@@ -1,5 +1,30 @@
 # bash completion for ant and phing                        -*- shell-script -*-
 
+_ant_parse_targets()
+{
+    local line basedir
+
+    basedir=$(dirname "$1")
```

Comment:
To be honest I do not understand that syntax. I am not Bash guru though. What does it mean?